tests/unit/DependencyInjection/DependencyInjection.class.test.js

```diff
@@ -44,4 +44,35 @@ describe('DependencyInjection/DependencyInjectionClass', () => {
       expect(requestService.di instanceof DependencyInjection).toEqual(true);
     });
   });
+
+  describe('isOffline', () => {
+    afterEach(() => {
```

Comment:
Just a thought, will these tests still work if I've set `USE_SERVERLESS_OFFLINE` in my terminal? It might be safer to add a `before` hook that deletes any pre-existing `process.env.USE_SERVERLESS_OFFLINE`.

Reply:
Good point!
82b8cff340b6189af851a395b207e00fee56eea6